src/main/java/uk/gov/hmcts/juror/api/bureau/service/BureauOfficerAllocatedRepliesServiceImpl.java

```diff
@@ -34,26 +29,31 @@ public BureauOfficerAllocatedResponses getBackLogData() {
         final List<User> bureauOfficers = Lists.newLinkedList(userRepository
             .findAll(UserQueries.activeBureauOfficers()));
 
+        Tuple backlogStats = jurorResponseRepository.getAssignRepliesStatistics();
+
         log.trace("Extracting backlog data");
         BureauBacklogCountData bureauBacklogCountDto = BureauBacklogCountData.builder()
-            .nonUrgent(bureauBacklogCountService.getBacklogNonUrgentCount())
-            .urgent(bureauBacklogCountService.getBacklogUrgentCount())
-            .allReplies(bureauBacklogCountService.getBacklogAllRepliesCount())
+            .nonUrgent(backlogStats.get(0, Integer.class).longValue())
```

Comment:
Would be better to use the query dsl names instead of index here from a readability viewpoint

Reply:
Updated 😄